src/ipa_pc/mod.rs

```diff
@@ -1072,11 +1150,12 @@ impl<G: AffineCurve, D: Digest> PolynomialCommitment<G::ScalarField> for InnerPr
 
         // h(X) polynomial added to the set of polynomials for multi-poly single-point batching
         let mut labeled_polynomials = labeled_polynomials;
+        let batch_degree = batch_polynomial.degree();
         let labeled_batch_polynomial = LabeledPolynomial::new(
             format!("Batch"),
             batch_polynomial,
             None,
-            None
+            Some(batch_degree)
```

Comment:
As Daniele probably already patched: set only hiding bound if hiding needed.

Reply:
Yes, this is done, you can see my comment above